validator/project_validator.go

```diff
@@ -1951,6 +1952,58 @@ func validateGenerateTasks(p *model.Project) ValidationErrors {
 	return validateTimesCalledPerTask(p, ts, evergreen.GenerateTasksCommandName, 1, Error)
 }
 
+func validateModuleUsageInGitGetProject(p *model.Project) ValidationErrors {
+	var errs ValidationErrors
+
+	bvs, _, err := bvsWithTasksThatCallCommand(p, "git.get_project")
+	if err != nil {
+		errs = append(errs, ValidationError{
+			Message: fmt.Sprintf("build variants could not be mapped to tasks that contain 'git.get_project' for project %s", p.Identifier),
+			Level:   Error,
+		})
+		return errs
+	}
+
+	for bvName, bv := range bvs {
```

Comment:
nit: I think this would be clearer if bvs here was named something `bvTaskCmds` or something to indicate that this is in a special format.

Reply:
renamed bv and bvs to be more descriptive on the data they hold!